components/eam/src/chemistry/modal_aero/modal_aero_data.F90

```diff
@@ -97,6 +99,13 @@ module modal_aero_data
          'fine_dust       ', &
          'coarse_seasalt  ', &
          'coarse_dust     '/)
+#elif ( defined MODAL_AERO_5MODE)
+    character(len=*), parameter :: modename_amode(ntot_amode) = (/ &
+         'accum           ', &
+         'aitken          ', &
+         'coarse          ', &
+         'primary_carbon  ', &
+         'strat_coarse  '/)
```

Comment:
The last element need to add additional space to have the same length as the other fields, otherwise gnu compiler would fail (string array constructor).

Reply:
@wlin7 thank you for finding it. I pushed the commit to remote